app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/DiffUtilCallback.java

```diff
@@ -0,0 +1,52 @@
+package com.futurice.freesound.feature.common.ui.adapter;
+
+import com.futurice.freesound.feature.common.DisplayableItem;
+
+import android.support.annotation.NonNull;
+import android.support.v7.util.DiffUtil;
+
+import java.util.List;
+
+final class DiffUtilCallback extends DiffUtil.Callback {
```

Comment:
Don't forget to add the standard copyright headers to the class files. Unfortunately, Apache v2.0 requires this. You should be able to achieve this automatically through the style files.